### PR TITLE
Fix some bugs

### PR DESCRIPTION
**Pull request by teageto**
I fixed the image in the private chat, when user has an avatar decoration don't show more double.

### CODE REVIEW

File: bd-scss.config.js
Comment (Gibbu) [inline bd-scss.config.js:1]: This is not necessary in this PR.

File: dist/RadialStatus.theme.css
Comment (Gibbu) [inline dist/RadialStatus.theme.css:1]: This is not necessary in this PR.

File: manifest.json
Comment (Gibbu) [inline manifest.json:1]: This is not necessary in this PR.

File: package-lock.json
Comment (Gibbu) [inline package-lock.json:1]: This is not necessary in this PR.

**Review by Gibbu — CHANGES_REQUESTED** [review]

**Review by teageto — COMMENTED** [review]
Sorry, I believed the another changes don't come to the pull request xD

**Review by Gibbu — APPROVED** [review]

**Pull request merged**